src/jltypes.c

```diff
@@ -3244,21 +3244,26 @@ void jl_init_types(void)
                                         jl_any_type, jl_null, 32);
 
     tv = jl_tuple1(tvar("T"));
+    jl_ref_type =
```

Comment:
@JeffBezanson should we explicitly disable gc in this `jl_init_types` function? it would be bad if it somehow ran, even though it shouldn't have managed to allocate that much memory by here